src/hooks/usePageTitlePriceUpdates.ts

```diff
@@ -1,29 +1,35 @@
 import { useEffect } from 'react';
 
+import { shallowEqual } from 'react-redux';
+
+import { SMALL_USD_DECIMALS } from '@/constants/numbers';
 import { DEFAULT_DOCUMENT_TITLE } from '@/constants/routes';
 
 import { useAppSelector } from '@/state/appTypes';
-import { getSelectedLocale } from '@/state/localizationSelectors';
 import {
+  getCurrentMarketConfig,
   getCurrentMarketId,
   getCurrentMarketMidMarketPrice,
   getCurrentMarketOraclePrice,
 } from '@/state/perpetualsSelectors';
 
+import { MustBigNumber } from '@/lib/numbers';
+
 import { useBreakpoints } from './useBreakpoints';
 
 export const usePageTitlePriceUpdates = () => {
-  const selectedLocale = useAppSelector(getSelectedLocale);
   const { isNotTablet } = useBreakpoints();
   const id = useAppSelector(getCurrentMarketId);
   const oraclePrice = useAppSelector(getCurrentMarketOraclePrice);
+  const { tickSizeDecimals } = useAppSelector(getCurrentMarketConfig, shallowEqual) ?? {};
+
   const orderbookMidMarketPrice = useAppSelector(getCurrentMarketMidMarketPrice);
 
   const price = orderbookMidMarketPrice ?? oraclePrice;
 
   useEffect(() => {
     if (id && price && isNotTablet) {
-      const priceString = price.toLocaleString(selectedLocale);
+      const priceString = MustBigNumber(price).toFixed(tickSizeDecimals ?? SMALL_USD_DECIMALS);
```

Comment:
Does this add commas to big numbers like bitcoin price?

Reply:
ah right - with go back to locale string, maybe theres a different setting

Reply:
lol looks like i need to set both min/max to match the decimal points